specs/windows/userassist.table

```diff
@@ -0,0 +1,12 @@
+table_name("userassist")
+description("UserAssist Registry Key tracks when a user executes an application from Windows Explorer.")
+schema([
+    Column("path", TEXT, "Application file path."),
+    Column("last_execution_time", TEXT, "Most recent time application was executed."),
+    Column("count", INTEGER, "Number of times the application was executed."),

```

Comment:
I could be wrong, but maybe this should read: `"Number of times the application has been executed."`?

Reply:
Fixed grammar